rootfs/standard/usr/bin/mynode_uninstall_app.sh

```diff
@@ -45,6 +45,8 @@ elif [ "$APP" = "joininbox" ]; then
     rm -rf /home/joinmarket/*
     rm -rf /home/joinmarket/joininbox-*
     rm -rf /home/joinmarket/.cache
+elif [ "$APP" = "jmg" ]; then
+    rm -rf /home/joinmarket/jmg
```

Comment:
This location will likely automatically get deleted if joininbox is re-installed. Moving it to /opy/mynode/jmg may be a better location.

Reply:
done